packages/common/src/api/createApi.ts

```diff
@@ -198,8 +220,9 @@ const buildEndpointHooks = (
       }
 
       const fetchWrapped = async () => {
-        if (cachedData || !context) return
-        if (status === Status.LOADING) return
+        if (!context) return
+        if ([Status.LOADING, Status.ERROR, Status.SUCCESS].includes(status))
```

Comment:
does this mean we can't refetch on error?

Reply:
yeah... for now. I believe @amendelsohn is working on an improvement